src/features/company/components/Company.tsx

```diff
@@ -3,31 +3,39 @@ import { Peers } from "../../peers";
 import { useSelector } from "react-redux";
 import { Loading } from "../../loading";
 import "./Company.css";
+import { GlobalState } from "../../../store";
 
 export const Company = () => {
   const { selectedCompanyOverview } = useSelector(
-    state => state.companyOverviewData
+    (state: GlobalState) => state.companyOverviewData
+  );
+  const { selectedTopPeers } = useSelector(
+    (state: GlobalState) => state.peerData
   );
-  const { selectedTopPeers } = useSelector(state => state.peerData);
 
   const renderCompanyComponent = () => (
     <>
       <div className="company-overview__title">
-        {selectedCompanyOverview.companyName || "N/A"} (
-        {selectedCompanyOverview.symbol || "N/A"})
+        {(selectedCompanyOverview && selectedCompanyOverview.companyName) ||
+          "N/A"}{" "}
```

Comment:
nit: I hate seeing `{" "}` for some reason :(
What about:
```ts
{`${(selectedCompanyOverview && selectedCompanyOverview.companyName) || 'N.A'} ${(selectedCompanyOverview && selectedCompanyOverview.symbol) || "N/A"}`}
```

Reply:
Good spot, will change

Reply:
👍 adding `{" "}` everywhere is maybe the ugliest thing _prettier_ does